docs/src/mixed_CPU_GPU.md

```diff
@@ -0,0 +1,45 @@
+# Mixed CPU and GPU computations
+
+If GPU memory is a limiting factor for your computation, it may be preferable to carry out particle operations on the CPU rather than on the GPU.
+This involves basically 4 steps:
+
+1) *At the top of the script*. The JustPIC backend must be set to CPU, in contrast with other employed packages (e.g. ParallelStencil):
+```julia
+const backend = JustPIC.CPUBackend 
+```
+
+2) *At memory allocation stage*. A copy of relevant CPU arrays must be allocated on the GPU memory. For example, phase ratios on mesh vertices:
+```julia
+phv_GPU = @zeros(Nc.x+1, Nc.y+1, Nc.z+1, celldims=(N_phases))
+```
+where `N_phases` is the number of different material phases and `@zeros()` allocates on the GPU.
+
+Similarly, GPU arrays must be copied to CPU memory:
+```julia
+V_CPU = (
+        x      = zeros(Nc.x+1, Nc.y+2, Nc.z+2),
+        y      = zeros(Nc.x+2, Nc.y+1, Nc.z+2),
+        z      = zeros(Nc.x+2, Nc.y+2, Nc.z+1),
+    )
+```
+where `zeros()` allocates on the CPU memory.
+
+4) *At each time step*. The particle will be stored on the CPU memory. It is hence necessary to transfer some information from the CPU to the GPU memory. For example, here's a transfer of phase proportions:
+
+```julia
+phv_GPU.data .= CuArray(phase_ratios.vertex).data
+```
+!!! we explicitly write `CuArray` - would be better to have something more explicit like `GPUArray` - is there such a thing?
```

Comment:
I guess the backend here depends either on what pkg one is using for the GPU backend abstraction, which does not match JPs backend now. With ParallelStencil this should be `Data.Array`, but this is different with KernelAbstractions. I would leave `CuArray`